src/test/librbd/test_fixture.cc

```diff
@@ -56,6 +56,14 @@ std::string TestFixture::get_temp_image_name() {
 }
 
 void TestFixture::SetUp() {
+  static bool seeded = false;
```

Comment:
Nit: perhaps move the logic into test_main and remove from here and test_librbd?